documentation/correctness.adoc

```diff
@@ -0,0 +1,123 @@
+= Model correctness
+
+We can identify a number of separate criteria for the correctness of a model.
+Failure to comply to one or more of these criteria doesn't imply incorrectness of the model.
```

Comment:
I do not understand this: if the criteria indicate correctness, I would expect that failure to comply should indicate that the model is incorrect _at same level_. It may be still usable, or maybe it could be "partially correct"? Or have some criteria a grade (info, warning) that indicates their severity?

Reply:
I reworded the 2nd sentence a bit. Instead of a grade, I mentioned "Can an incorrectness be recovered from (and if so: how)?"